### ISSUE TITLE
General front-end redesign

### ISSUE THREAD

**Issue by Ben-G-Man** [Taks0708/SOFTENG310-FinTrack#178]
**Is your feature request related to a problem? Please describe.**
After adding more content to the home page the front-end is now unintuitive and features duplicate elements.

**Describe the solution you'd like**
Elements the user might want to access from anywhere in the application should be locked to a section of the front-end which holds its position in the screen. Something like the following design should be implemented.

![image](https://github.com/user-attachments/assets/1eda857b-116a-473b-8094-4e302f511ab8)

**Comment by LX6T** [comment]
Still waiting on another PR from @tom1dev
